frontend/institution/descriptionInst/editDescriptionController.js

```diff
@@ -0,0 +1,27 @@
+(function(){
+    const app = angular.module('app');
+
+    app.controller("EditDescriptionController", function EditDescriptionController(institution, InstitutionService,
+        ObserverRecorderService, $rootScope, $mdDialog){
+        const descriptionCtrl = this;
+        let observer;
+    
+        descriptionCtrl.$onInit = () => {
+            descriptionCtrl.institution = institution;
+            observer = ObserverRecorderService.register(descriptionCtrl.institution);
+        };
+
+        /** Save changes of institution and emit event. 
+         */
+        descriptionCtrl.save = () => {
+            const patch = ObserverRecorderService.generate(observer);
```

Comment:
I'm not sure about how this service works, but shouldn't u unregiste the observer after that? Maybe this observer leaves even when this controller dies...